contracts/neutron_interchain_queries/src/contract.rs

```diff
@@ -183,6 +201,28 @@ pub fn register_transfers_query(
     Ok(Response::new().add_message(msg))
 }
 
+pub fn register_cw20_balance_query(
+    connection_id: String,
+    update_period: u64,
+    cw20_contract_address: String,
+    account_address: String,
+) -> NeutronResult<Response<NeutronMsg>> {
+    // cw_storage_plus uses this prefix for maps
+    let mut storage_key = vec![0u8, 7u8];
+
+    storage_key.extend_from_slice("balance".as_bytes());
+    storage_key.extend_from_slice(account_address.as_bytes());
```

Comment:
This stuff definitely should be in the SDK helpers not here

Reply:
I think Mike means that it's not only `new_register_wasm_contract_store_query_msg` should be in the helpers package as the most generic helper for interacting with contract's storage, but a more specific helper to register cw20 balance query as well (e.g. `new_register_cw20_balance_query_msg` analogically to `new_register_delegator_delegations_query_msg`). the helper can have contract and holder addresses as parameters and assemble the needed key inside with these `vec![0u8, 7u8]` and `balance` reused from constants defined here: https://github.com/neutron-org/neutron-sdk/blob/main/packages/neutron-sdk/src/interchain_queries/v045/types.rs

Reply:
but now I wonder if we need such a helper — to query a cw20 balance

Reply:
I am very hesitant creating even some kind of "general helper which works for canonical cw20-base contract" because users will definitely try to use it and it won't work in some cases.

Reply:
> I could implement some kind of storage key factory like this:

To be honest it doesn't make anything easier to use, I think I should abandon this idea.